src/packages/auth/src/authentication/apollo/plugin.ts

```diff
@@ -25,6 +25,8 @@ const didEncounterForbiddenError = (error: any): error is ForbiddenError =>
 	error.extensions.code === ErrorCodes.FORBIDDEN;
 const didEncounterChallengeError = (error: any): error is ChallengeError =>
 	error.extensions.code === ErrorCodes.CHALLENGE;
+const didEncounterRestrictedFieldError = (error: any): error is RestrictedFieldError =>
+	error.extensions?.isRestrictedFieldError;
```

Comment:
Should we put the `?` on the other ones above this for good measure while we're in here?